x/storage/types/message.go

```diff
@@ -181,6 +203,19 @@ func (msg *MsgCreateBucket) ValidateBasic() error {
 	return nil
 }
 
+func (msg *MsgCreateBucket) ValidateRuntime(ctx sdk.Context) error {
+	if ctx.IsUpgraded(upgradetypes.Eddystone) {
+		if len(msg.Tags.GetTags()) > MaxTagCount {
```

Comment:
Please verify the `key` and `value` length in the `validateBasic` function for all msg that have tags

Reply:
Done